src/material-examples/button-card/button-card-example.html

```diff
@@ -0,0 +1,9 @@
+<mat-card>
+  <mat-card-header>
+    <mat-card-title>Card with action buttons</mat-card-title>
+  </mat-card-header>
+  <mat-card-actions align="end">
+    <button mat-button>CANCEL</button>
+    <button mat-button color="primary">ACCEPT</button>
```

Comment:
These should have an `aria-label` along the lines of "Example button, no action"

(here and elsewhere, where the label can be expanded with what the button is an example of)